src/main/flight/position.c

```diff
@@ -66,6 +66,8 @@ static int16_t estimatedVario = 0; // in cm/s
 
 void positionInit(void)
 {
+    positionControlInit();
```

Comment:
Good point.  Initialised independently.
Also ensured that GPS Rescue and Alt Hold are initialised after position_control since they needs its PID values.
Hopefully did that correctly.

Reply:
not required now we use direct access